docs/source/adb.rst

```diff
@@ -0,0 +1,131 @@
+.. _adb:
+
+####################
+Android Debug Bridge
```

Comment:
Considering that the title is also used now in links (due to the .. _adb: bit of code, it wouldn't quite fit stylistically.